packages/eslint-plugin-office-addins/src/rules/load-object-before-read.ts

```diff
@@ -37,6 +37,20 @@ export = {
       return false;
     }
 
+    function hasBeenLoaded(
+      node: TSESTree.Node,
+      loadLocation: Map<string, number>,
+      propertyName: string
+    ): boolean {
+      if (
+        loadLocation.has(propertyName) && // If reference came after load, return
+        node.range[1] > (loadLocation.get(propertyName) ?? 0)
+      ) {
+        return true;
+      }
+      return false;
+    }
```

Comment:
Can just return the result of the if statement directly . . . no need for the expanded if statement.